docker/include/bin/start_faux

```diff
@@ -86,6 +86,22 @@ if [ -n "${options[xdhcp]}" ]; then
         ip addr add 10.20.0.5/16 dev $intf_name broadcast 10.20.255.255
     fi
     ip addr show $intf_name
+    route add default gw 10.20.0.1
```

Comment:
static ip device is updated to have DNS and NTP traffic in my newest PR. Please merge master.